backend/src/main/java/com/webappsbusters/parcelmanagement/domain/ParcelSize.java

```diff
@@ -0,0 +1,8 @@
+package com.webappsbusters.parcelmanagement.domain;
+
+public enum ParcelSize {
+    None,
+    Small,
+    Medium,
+    Large
```

Comment:
Enums should have elements in UPPERCASE - NONE, SMALL, ...

Reply:
done